Cmake trouble

Following the instructions here https://github.com/vgough/encfs/wiki/Installing-Encfs on ubuntu 14.04 in lxc I run into the following error on cmake ..

`-- Enabled syslog logging support
CMake Warning at CMakeLists.txt:131 (find_package):
  By not providing "FindIntl.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Intl", but
  CMake did not find one.

  Could not find a package configuration file provided by "Intl" with any of
  the following names:

```
IntlConfig.cmake
intl-config.cmake
```

  Add the installation prefix of "Intl" to CMAKE_PREFIX_PATH or set
  "Intl_DIR" to a directory containing one of the above files.  If "Intl"
  provides a separate development package or SDK, be sure it has been
  installed.

CMake Error at /usr/share/cmake-2.8/Modules/FindPackageHandleStandardArgs.cmake:108 (message):
  Could NOT find Gettext (missing: GETTEXT_MSGMERGE_EXECUTABLE
  GETTEXT_MSGFMT_EXECUTABLE)
Call Stack (most recent call first):
  /usr/share/cmake-2.8/Modules/FindPackageHandleStandardArgs.cmake:315 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-2.8/Modules/FindGettext.cmake:58 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  po/CMakeLists.txt:1 (find_package)

-- Configuring incomplete, errors occurred!`

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -46,6 +46,8 @@ EncFS depends on a number of libraries:
     * tinyxml2 : for reading and writing XML configuration files
     * gettext : internationalization support
     * libintl : internationalization support
+    * cmake : version 3.2 or newer
+    * GNU make or ninja-build : to run the build for cmake
 
 Compiling on Debian and Ubuntu
 ==============================
@@ -54,3 +56,6 @@ See the automated build static in README.md for current build status on Ubuntu s
 
 The build configuration files (circle.yml) always contains up-to-date
 instructions to build EncFS on Ubuntu distributions.
+
+On Debian Stable, additional installations from Backports branch might be
+required (cmake 3.x for example, see https://backports.debian.org/ for instructions).

